.github/actions/ts-e2e/action.yml

```diff
@@ -29,15 +29,19 @@ runs:
     - name: cargo build
       if: env.s3_file_exist == '' # if empty, we have not built and uploaded this binary to s3 yet
       run: |
-        cargo build --bin sui --features indexer
+        if [[ "${{ inputs.ref }}" == 'devnet' || "${{ inputs.ref }}" == 'testnet' ]]; then
```

Comment:
Note that this is a workaround until we get this code into devnet/testnet branches. When that happens, we need to remove this (a.k.a the workflow will fail anyway).